packages/reference/src/entries/WrappedEntryCard/FetchingWrappedEntryCard.tsx

```diff
@@ -132,20 +133,34 @@ export function FetchingWrappedEntryCard(props: EntryCardReferenceEditorProps) {
       onEdit,
       onRemove,
     };
+
+    function renderDefaultCard(customProps?: Partial<WrappedEntryCardProps>) {
+      const builtinCardProps: WrappedEntryCardProps = {
+        ...sharedCardProps,
+        // eslint-disable-next-line react/prop-types
```

Comment:
at these line eslint is complaining:
```
  140:35  error  'hasCardEditActions' is missing in props validation                   react/prop-types
  141:25  error  'sdk' is missing in props validation                                  react/prop-types
  141:29  error  'sdk.space' is missing in props validation                            react/prop-types
  141:35  error  'sdk.space.getAsset' is missing in props validation                   react/prop-types
  142:42  error  'sdk' is missing in props validation                                  react/prop-types
  142:46  error  'sdk.space' is missing in props validation                            react/prop-types
  142:52  error  'sdk.space.getEntityScheduledActions' is missing in props validation  react/prop-types
```

Any idea how to prevent that?